content/blog/2022/2022-02-11-edit-image-collection.md

```diff
@@ -0,0 +1,91 @@
+
+*Our server delete all images immediately after embedding. If you would not like to share your photos with our server anyway, you can select the SqueezeNet embedder that embeds images at your computer.*
+
+{{< window-screenshot src="/blog_img/2022/2022-02-11-image-embedding.jpg" >}}
+
+The Image Embedding widget added 2048 columns with numbers to each image in the data table. Those numbers describe properties that neural networks inferred from pictures. Numbers are not understandable to humans but make sense to machine learning algorithms.
+
+### Image grid will place similar photographs closer on the grid
+
+[The Image Grid](/widget-catalog/image-analytics/imagegrid/) uses [t-sne embedding](/widget-catalog/unsupervised/tsne/) to project images on 2-dimensional plane based on their image embeddings. More similar images will be closer in the plot. 
+
+We passed embedded images from the Image Embedding to the Image Grid widget.
+
+{{< window-screenshot src="/blog_img/2022/2022-02-11-image-grid.jpg" >}}
+
+Image Grid has placed similar images closer in the grid. For example, in the top right corner are images of food, in the top left corner are photos of cities, on the right side are pictures of mountains, and at the bottom left are images of sunsets. The Image Grid had more difficulties with snowy photos. Some are placed at the bottom (they seem to include some mountain image elements), and three central ones seem to have elements of cities, mountains and snow.
+
+Since images that belong to the same group are close together, it is easy to mark groups of images manually (colours around images). Marked groups are in the output table as class attributes. We can save them on the computer such that each group is a separate folder with [the Save Images widget](/widget-catalog/image-analytics/saveimages).
+
+**Related:** {{< link_new url="/blog/2017/04/03/image-analytics-clustering/" name="Image Analytics: Clustering">}}
+
+### Training the model to sort images
+
+The Image Grid is excellent to discover similar images. Still, it is not practical to sort photos into category groups. If we would like to do that more often, we would always need to assign images to groups manually. Can we have a pipeline that would allow us to make a model that we can reuse later when we take new photos s?
+
+**Related**: {{< link_new url="https://www.youtube.com/watch?v=lvgx62a8XQk&list=PLmNPvQr9Tf-aRstY69vGAPO_c5RaceBhN&index=1&ab_channel=OrangeDataMining" name="Video on image classification">}}
+
+We can train a classification model that will mark images with categories. 
+
+We prepared a separate training dataset with pictures in 5 subfolders. Folders are named: city, food, mountains, sunsets and winter. Names of the flowers define categories; once the model is trained, it will always classify images in those categories.
```

Comment:
Names of the flowers?

Reply:
Haha. This one is probably Grammarly's correction that I didn't notice.